KubeArmor/feeder/feeder.go

```diff
@@ -322,6 +322,23 @@ func NewFeeder(node *tp.Node) *Feeder {
 	}
 	fd.Listener = listener
 
+	if cfg.GlobalCfg.GRPC == "0" {
+		pidFile, err := os.Create(cfg.PIDFilePath)
+		if err != nil {
+			kg.Errf("Failed to create file %s", cfg.PIDFilePath)
+			return nil
+		}
+
```

Comment:
```suggestion
		pidFile, err := os.Create(cfg.PIDFilePath)
		if err != nil {
			kg.Errf("Failed to create file %s", cfg.PIDFilePath)
			return nil
		}
		
		defer pidFile.Close()
```
I believe we should close the file after writing to it.